.github/workflows/pull-latest-docs.yml

```diff
@@ -0,0 +1,24 @@
+name: Pull Talawa API Changes
+
+on:
+  schedule:
+    - cron: '*/5 * * * *' 
```

Comment:
Not every 5 minutes, but weekly. 
@palisadoes What is your suggestion? Daily, Weekly or Monthly.